app/validators/number.js

```diff
@@ -12,6 +12,10 @@ const {
 
 export default Base.extend({
   validate(value, options) {
+    if (options.allowBlank && isEmpty(value)) {
```

Comment:
To be consistent with the rest of the validators, can you move this block below the variable declaration